misc/msvs/vcxproj.template

```diff
@@ -10,16 +10,17 @@
     <VCProjectUpgraderObjectName>NoUpgrade</VCProjectUpgraderObjectName>
   </PropertyGroup>
   %%PROPERTIES%%
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="Configuration">
     <ConfigurationType>Makefile</ConfigurationType>
     <UseOfMfc>false</UseOfMfc>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>%%PlatformToolset%%</PlatformToolset>
+    <DefaultPlatformToolset>$(PlatformToolset)</DefaultPlatformToolset>
     <OutDir>$(SolutionDir)\bin\$(GodotPlatform)\$(GodotConfiguration)\</OutDir>
     <IntDir>obj\$(GodotPlatform)\$(GodotConfiguration)\</IntDir>
     <LayoutDir>$(OutDir)\Layout</LayoutDir>
   </PropertyGroup>
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" Condition=" '$(PlatformToolset)' != 'Clang' "/>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" Condition=" '$(PlatformToolset)' != 'Clang' "/>
```

Comment:
This should not exclude the clang toolset. Is this a Rider thing? There are clang toolsets for VS and this will break all of the platforms that use the clang toolset.

Reply:
Right, those come with VS and would always be available with VS, but not necessarily in other IDEs. Checking for their existence as a condition should be enough to make all the IDEs happy.

Reply:
I have considered the `exists` check and preliminary decided against it not to hide the error for users, who want to use MSVC, but haven't installed it. Now I added one for the Microsoft.Cpp.Default.props and will see about others.